protoutils/protoutils_test.go

```diff
@@ -332,24 +332,33 @@ func TestStructToStructPb(t *testing.T) {
 	}
 }
 
+func TestStructToStructPbOmitEmpty(t *testing.T) {
+	expected := map[string]interface{}{"x": 0.0, "y": 0.0}
+	t.Run("StructToStructPBIgnoreOmitEmpty includes all values", func(t *testing.T) {
```

Comment:
[nit] Just an FYI that it's okay to not run this as a subtest (within t.Run) since you only have logic for one test in this function. t.Run just lets you differentiate between different subtests within a given Test function (and do useful things like run them in parallel, in isolation, etc.). If you did this in anticipation of adding future tests or to have a more descriptive subtest name then it's okay to keep it the way you have it here, just wanted you to be aware!

Reply:
Ah makes sense, ill change it.